src/api/generate-csv/map-application-to-csv/helpers/map-month-string/index.ts

```diff
@@ -0,0 +1,15 @@
+/**
+ * mapMonthString
+ * Map credit period answer to a month/s string
+ * @param {Number} Credit period
+ * @returns {String} Credit period with month/s string
+ */
+const mapMonthString = (answer: number) => {
+  if (answer === 1) {
```

Comment:
Ternary operator will reduce number of line.

Reply:
I think a one liner makes it harder to read but have made the change.